DesignPatterns/TemplateMethod/Article/TemplateMethodArticleTests.swift

```diff
@@ -0,0 +1,75 @@
+import Testing
+
+@testable import TemplateMethodArticleCode
+
+@Test func testSeries10AppleWatchProcess() {
+    let configuration = MockSeries10AppleWatchConfiguration()
+    configuration.process()
+
+    #expect(configuration.operationOrder.count == 5)
+    #expect(configuration.operationOrder[0] == "mapDataForWatchCaseSize")
+    #expect(configuration.operationOrder[1] == "mapDataForWatchCaseMaterial")
+    #expect(configuration.operationOrder[2] == "mapDataForWatchBand")
+    #expect(configuration.operationOrder[3] == "mapDataForWatchBandSize")
+    #expect(configuration.operationOrder[4] == "mapDataForWatchEngraving")
+}
+
+@Test func testHermèsSeries10AppleWatchProcess() {
+    let configuration = MockHermèsSeries10AppleWatchConfiguration()
+    configuration.process()
+
+    #expect(configuration.operationOrder.count == 5)
+    #expect(configuration.operationOrder[0] == "mapDataForWatchCaseSize")
+    #expect(configuration.operationOrder[1] == "mapDataForWatchCaseMaterial")
+    #expect(configuration.operationOrder[2] == "mapDataForWatchBand")
+    #expect(configuration.operationOrder[3] == "mapDataForWatchBandSize")
+    #expect(configuration.operationOrder[4] == "mapDataForWatchEngraving")
+}
+
+class MockSeries10AppleWatchConfiguration: Series10AppleWatchConfiguration {
```

Comment:
This isn't a mock but a stub. Good to get clarity on these two, though most devs use the word interchangeably. But let's change that by modelling best naming practices in our code. 

Series10AppleWatchConfigurationStub is the right name. 

You'll need to update this in all the PR's please.